apps/emqx/src/emqx_channel.erl

```diff
@@ -1572,6 +1575,47 @@ enrich_client(ConnPkt, Channel = #channel{clientinfo = ClientInfo}) ->
             {error, ReasonCode, Channel#channel{clientinfo = NClientInfo}}
     end.
 
+initialize_client_attrs_from_cert(
+    #{
```

Comment:
Do we purposely limit number of cert attrs to one?

Reply:
we can change the schema to allow many in the future if needed.